hr_payroll_commission/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Payroll Commissions and Reinbursements
+======================================
+
+This module allows you to add commission and reimbursement to payslip
+
+**Features list :**
+    * Add Worked Hours to contract.
+    * Add Hourly Rate to contract.
+    * Compute the Wage of contract, based on Worked Hours and Hourly Rate.
+    * Add commision rate, commision and reimbursement to contract.
+    * Link Expenses and Invoices to Payslip.
+
+**For further information:**
+    * Commissions and reimbursements management: http://open-net.ch/blog/la-comptabilite-salariale-suisse-avec-odoo-1/post/salaires-avec-odoo-commissions-et-notes-de-frais-78
+

```

Comment:
We should have a better explanation here.
In particular, I'm not familiar with the link between Commissions and Expenses. 
Maybe that's something specific to some countries.

I'm familiar with expense reimbursements paid on Payslips, but in my experience that has nothing to do with Commissions.

For this to be a universal feature, could you please expand a little on the use cases?